app/src/main/java/com/space/quiz_app/presentation/feature/base/fragment/QuizBaseFragment.kt

```diff
@@ -39,6 +42,7 @@ abstract class QuizBaseFragment<VM : QuizBaseViewModel> : Fragment() {
         super.onViewCreated(view, savedInstanceState)
         onBind()
         observeNavigation()
+        observeErrorState()
         findNavController().popBackStack()
```

Comment:
amas yovel prze dagikomentareb, ver vibe rato weria findNavController().popBackStack() onViewCreated - shi

Reply:
done, deleted and replaced with fragment finish in home fragment